doc/source/user_guide/external_records.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_grantami_bomanalytics_external_record_references:
+
+Using external records in analysis
+==================================
+
+.. versionadded:: 2.4
+
+Introduction
+------------
+
+MI Restricted Substances and Sustainability Reports 2026 R1 introduces the ability to perform analysis on records stored
+in separate databases to the Restricted Substances and Sustainability database. These records are referred to below as
+'external records', and the databases they are stored in are referred to as 'external databases'.
+
+Separating data across databases in this way can simplify data management by making it easier to share responsibility
+and ownership for different datasets across functional groups. Additionally, it is easier to adopt the Restricted
+Substances and Sustainability solution if existing data can be easily linked to representative data supplied with the
+Restricted Substances and Sustainability database.
+
+When a BoM Analytics query is submitted that includes external records, Granta MI first identifies equivalent Restricted
+Substances and Sustainability records by following cross-database links between the Restricted Substances and
+Sustainability records and the external records. The results include both the Restricted Substances and Sustainability
+records and the external records, providing clear traceability for how the results were determined.
+
+This page describes the pre-requisites for performing analysis based on external records, and provides code examples for
+common use cases.
+
+.. _ref_grantami_bomanalytics_external_record_references_pre_reqs:
+
+
+Pre-requisites
+--------------
+
+The following conditions must be met to use this feature:
+
+* The version of ``grantami-bomanalytics`` must be version 2.4 or later.
+* The Granta MI server is running MI Restricted Substances and Sustainability Reports 2026 R1 or later.
+* The records used for analysis are linked to equivalent records in the main Restricted Substances and Sustainability
+  database. See Ansys help for more details.
```

Comment:
Created https://github.com/ansys/grantami-bomanalytics/issues/892 to track adding this link in once ready.